JMeterAKSPipeline/CreateLoadTestInfra.ps1

```diff
@@ -0,0 +1,47 @@
+param(
+    [Parameter(Mandatory = $True)]
+    [string]$tenant,
+    [Parameter(Mandatory = $True)]
+    [string]$defaultNamespace,
+    [Parameter(Mandatory = $True)]
+    [string]$resourceGroup,
+    [Parameter(Mandatory = $True)]
+    [string]$aksClusterName,
+    [Parameter(Mandatory = $True)]
+    [string]$spnClientId,
+    [Parameter(Mandatory = $True)]
+    [string]$spnClientSecret,
+    [Parameter(Mandatory = $True)]
+    [string]$aksRegion,
+    [Parameter(Mandatory = $False)]
+    [string]$nodeVmSize = "Standard_D2s_v3"
+)
+
+function log([string] $message, [string] $color) {
+    Write-Host "$(get-date) $message"   -ForegroundColor $color 
+    Write-Host " " 
```

Comment:
why we have blank content for write-host here ?

Reply:
It is there to print an empty line.
I just tried without it, and have to say it is much easier to read it with these line gaps.